roles/fs-drift/tasks/main.yml

```diff
@@ -59,7 +59,7 @@
             spec:
               containers:
                 - name: publisher-container
-                  image: quay.io/cloud-bulldozer/fs-drift:master
+                  image: quay.io/cloud-bulldozer/fs-drift:latest
```

Comment:
Can we update this line to be in line w/ the others to allow you to overwrite the image. ie:
```
image: "{{ workload_args.image | quay.io/cloud-bulldozer/fs-drift:latest }}"
```